php/Blocks/Loader.php

```diff
@@ -158,6 +166,16 @@ public function editor_assets() {
 			]
 		);
 
+		if ( Settings::ANALYTICS_OPTED_IN_VALUE === get_option( Settings::ANALYTICS_OPTION_NAME ) ) {
+			wp_enqueue_script(
+				self::ANALYTICS_SCRIPT_SLUG,
+				'https://www.googletagmanager.com/gtag/js?id=UA-17364082-14',
```

Comment:
The UA id should probably be unique to GCB

Reply:
Should be able to get that from the GA dashboard

Reply:
OK nice, could you Slack me a link to that?